test/util/genesis/accounts.go

```diff
@@ -20,40 +19,28 @@ const (
 	DefaultInitialBalance = 1e15 // 1 billion TIA
 )
 
-// Account represents a user account on the Celestia network.
+// KeyringAccount represents a user account on the Celestia network.
```

Comment:
Perhaps this part of the comment does not apply anymore:
```
or an address needs to be provided
```

Reply:
[suggestion] The name account in `KeyringAccount` is a bit confusing, as it is more of some info about the account not the actual account. I'd suggest using an alternative name  like `KeyringAccountInfo`.

Reply:
So I tried to distinguish `KeyringAccount` as an `Account` that is paired with a Keyring belonging to the `Genesis` struct. If you don't want to use the keyring provided by the `Genesis` struct then you use an `Account` and therefore must include the public key

Reply:
Thanks, your explanation has made things clearer. I think it would be helpful to also add these explanations as a comment to both structs.